packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -19,7 +20,7 @@ async function getValidOriginsListFromCDN(): Promise<string[]> {
     return validOriginsCache;
   }
   if (!inServerSideRenderingEnvironment()) {
-    return fetch(validOriginsCdnEndpoint)
+    return fetch(validOriginsCdnEndpoint, { signal: AbortSignal.timeout(ORIGIN_LIST_TIMEOUT) })
       .then((response) => {
         if (!response.ok) {
```

Comment:
Can we add a log statement in the `then` call to indicate that `fetch` has completed?

The idea is that between the two new log statements I was requesting, you could tell how long this call is taking by reading the logs.